lib/v2/gov/huizhou/zwgk/index.js

```diff
@@ -0,0 +1,64 @@
+const got = require('@/utils/got');
+const config = require('@/config').value;
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+const rootURL = 'http://www.huizhou.gov.cn';
+
+module.exports = async (ctx) => {
+    const cate = ctx.params.category ?? 'zwyw';
+    const url = `${rootURL}/zwgk/hzsz/${cate}`;
+
+    const response = await got.get(url, {
+        headers: {
+            UserAgent: config.ua,
+        },
```

Comment:
RSSHub will add this header. No need to do it twice
https://github.com/DIYgod/RSSHub/blob/a652e7bf308a84d990511f5642297fb464dd96a4/lib/utils/got.js#L33-L35